erpnext_com/www/docs/user/manual/en/stock/quick-stock-balance.md

```diff
@@ -0,0 +1,23 @@
+<!-- add-breadcrumbs -->
+# Quick Stock Balance
+
+Quick Stock Balance is a tool that can be used to quickly retrieve the current stock balance (quantity) and value of an item, in the specified warehouse, on the specified date.
+
+It comes in handy during audits, while verifying stock details before transactions, etc.
+
+To access this tool, go to:
+> Home > Stock > Tools > Quick Stock Balance
```

Comment:
THIS BREADCRUMB DOES NOT EXIST IN THE PRODUCT

Reply:
this was discussed :)